modules/shared/src/main/scala/org/tessellation/json/JsonSerializer.scala

```diff
@@ -0,0 +1,13 @@
+package org.tessellation.json
+
+import io.circe.jawn.JawnParser
+import io.circe.syntax._
+import io.circe.{Decoder, Encoder}
+
+object JsonSerializer {
+  def serialize[A: Encoder](content: A) =
+    content.asJson.noSpaces.getBytes("UTF-8")
+
+  def deserialize[A: Decoder](content: Array[Byte]): Either[Throwable, A] =
+    JawnParser(false).decodeByteArray(content)
```

Comment:
Specifying charset is not necessary?

Reply:
There is no parameter to pass charset, and I've checked, that under the hood utf-8 is used.